projects/storyteller/webapp/src/app/core/services/firebase.service.ts

```diff
@@ -1,21 +1,21 @@
-import { Injectable } from '@angular/core';
-import { AngularFirestore, AngularFirestoreCollection } from 'angularfire2/firestore';
+import { Injectable, OnInit } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection, AngularFirestoreDocument } from 'angularfire2/firestore';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
-
-import { StoryList, Screenshot } from '@/core/proto';
+import { StoryList, Screenshot, ReviewerConfig } from '@/core/proto';
 import { EncodingService } from './encoding.service';
 import { AngularFireAuth } from 'angularfire2/auth';
 import * as firebase from 'firebase/app';
+
 interface FirebaseElement {
   proto: string;
 }
 
 @Injectable()
 export class FirebaseService {
+
```

Comment:
It's just question of style guide. Could you remove empty line here?

Reply:
done

Reply:
Are you sure it's done?